docs/qcfractal/source/managers_faq.rst

```diff
@@ -0,0 +1,130 @@
+Queue Manager Frequent Questions and Issues
+===========================================
+
+This page documents some of the frequent questions and issues we see with the
+Queue Managers. If this page and none of the other documentation pages have
+answered your question, please ask `on GitHub <https://github.com/MolSSI/QCFractal/>`_ or
+`join our Slack group <https://join.slack.com/t/qcdb/shared_invite/enQtNDIzNTQ2OTExODk0LWM3OTgxN2ExYTlkMTlkZjA0OTExZDlmNGRlY2M4NWJlNDlkZGQyYWUxOTJmMzc3M2VlYzZjMjgxMDRkYzFmOTE>`_
+to get assistance.
+
+Common Questions
+----------------
+
+How do I get more information from the Manger?
+++++++++++++++++++++++++++++++++++++++++++++++
+
+Turn on ``verbose`` mode, either add the ``-v`` flag to the CLI, or set the
+``common.verbose`` to ``True`` in the YAML file. Either will make your logger much more loud.
```

Comment:
the `-v`/`-vv`/`-vvv` for verbose/debug/trace modes is easy to remember and something to consider.